Resources/doc/anonymizer.md

```diff
@@ -0,0 +1,178 @@
+# Anonymizer
```

Comment:
- Anonymizer type Array is missing
- Do we want to say anything about the unique nature of ObjectAnonymizer and CollectionAnonymizer? I personally found that - without knowing the architecture of the bundle - I got confused because their implementation is different from the others

Reply:
The collection type is the ArrayAnonymizer - the examples explain the working of the type in detail.